config/StoredProcedures/GetInventoryLocations.sql

```diff
@@ -0,0 +1,12 @@
+DROP PROCEDURE IF EXISTS GetInventoryLocations;
+
+DELIMITER $$
+CREATE PROCEDURE GetInventoryLocations
+()
+BEGIN
+
+SELECT *
+FROM Location;
+
+END $$
+DELIMITER ;
```

Comment:
Try SELECT DISTINCT(Location) FROM Piles;  instead of making a new table that needs to be updated in a number of routes